addrmgr/addrmanager.go

```diff
@@ -1028,6 +1029,25 @@ func (a *AddrManager) HasLocalAddress(na *wire.NetAddress) bool {
 	return ok
 }
 
+// FetchLocalAddresses fetches a summary of local addresses information for
+// the getnetworkinfo rpc.
+func (a *AddrManager) FetchLocalAddresses() []dcrjson.LocalAddressesResult {
```

Comment:
I would prefer to avoid adding dependencies on `dcrjson`.  The goal is to reduce dependencies as opposed to adding more.  I'd suggest creating a new type to return and then the `rpcserver` would convert it accordingly.